### ISSUE TITLE
[ACCESSIBILITY] - Large text not reflowing correctly

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#511]
**Android 10/iOS 14.4**

**Description**
When font sizes are increased on the Android/iOS screen, some screens do not reflow the text so that the large font is visible. It is not possible to move the screen to the right to see this text. This is a Priority 1 issue, it must be fixed.

**Steps to reproduce the behavior:**

1. In Accessibility Settings, set font size to largest available
2. Go to Events Listing screen (or Browse)
3. Upon landing user can see that the Events filter buttons and dropdowns are cutoff 
4. They do not reflow so they are visible and user is unable to interact with them

Expected behavior
Screen text and buttons must reflow to adapt to large screen fonts.

**Smartphone** (please complete the following information):
**Device**: Google Pixel 4a/ iPhoneXR
**OS**: Android 10/ iOS 14.4

**Screenshots**
See below.
![reflow PNG](https://user-images.githubusercontent.com/82892873/125797963-cb275dff-8edf-4abf-af41-9a7f9efc6297.png)

**Comment by acovher2** [comment]
Hello, 
It has been 1 month; are there any updates on this issue? 
Please provide any information you can. Thank you.

**Comment by TodorBachvarov** [comment]
If we want to remove the horizontal scrolling probably the whole panel should be redesigned. When we are improving the Large Text Accessibility support we have two options: 
First is to stretch the widget in width and use a horizontal scrolling. 
Second is to stretch the widget in height. (which takes much more space and in most cases vertical scrolling).
For this panel the filters and tabs are outside of the scrolling area and for phones with small screens the user won't be able to see even one event (the scrollable area will become too small). And we cannot add the filters into the scrollable area easily because we are using our custom filter widget which is designed to be placed at same screen position.
Also in this panel we have another vertical scrolling for the Composite Events.

**Comment by TodorBachvarov** [comment]
The horizontal scrolling for tabs and filters is removed.

**Comment by acovher2** [comment]
Should this be seen in v2.6.21?

**Comment by mihail-varbanov** [comment]
@acovher2, it should be seen in 2.6.25.

**Comment by acovher2** [comment]
You were not lying.... widget takes up the whole screen now.  I'm asking advice what to do next as I'm not sure this is viable.

**Comment by acovher2** [comment]
Please set device display to largest and review app with this set.  Many screens have content unreadable and or functionality obscured. Please address.

**Comment by TodorBachvarov** [comment]
@acovher2 This issue is pretty old and I assume that the original request (to remove the horizontal scrolling) was fixed long ago. Since there is no approved redesign for this panel, there is nothing that I can do. Could we close this ticket and try creating different ticket for each specific problem (panel/screen) because they will require individual approach and probably some redesign which should be discussed separately.
